TicketingService.Monitoring/Handlers-ByStatus.cs

```diff
@@ -0,0 +1,32 @@
+namespace TicketingService.Monitoring;
+
+using Abstractions;
+using Marten;
+
+public static partial class Handlers
+{
+    public static async Task<IResult> GetList(
+        IDocumentStore store,
+        string? fromDate,
+        string? toDate,
+        string? statuses,
+        CancellationToken cancellationToken)
+    {
+        var tickets = await store.QueryTickets(
+            fromDate,
+            toDate,
+            statuses,
+            new List<TicketStatus> {TicketStatus.Pending, TicketStatus.Created},
```

Comment:
I would not make 2 parameters, but one. You can see that in a different case you sent `null`.